go/ql/lib/semmle/go/dataflow/internal/FlowSummaryImplSpecific.qll

```diff
@@ -56,6 +56,9 @@ DataFlowType getCallbackParameterType(DataFlowType t, int i) { none() }
  */
 DataFlowType getCallbackReturnType(DataFlowType t, ReturnKind rk) { none() }
 
+/** Gets the type of synthetic global `sg`. */
+DataFlowType getSyntheticGlobalType(SummaryComponent::SyntheticGlobal sg) { any() }
```

Comment:
This might be a problem re: previous comments about Go's DataFlowType being equal to `Type` while other languages that use an `any()` stub tend to have a unit `DataFlowType`.